src/Temporalio/Worker/WorkflowInstance.cs

```diff
@@ -1715,6 +1732,7 @@ public override Task DelayAsync(DelayAsyncInput input)
                         {
                             Seq = seq,
                             StartToFireTimeout = Google.Protobuf.WellKnownTypes.Duration.FromTimeSpan(delay),
+                            Summary = instance.PayloadConverter.ToPayload(string.Empty),
```

Comment:
```suggestion
```

I think we want this to be null instead of a full payload (and we will add if summary is present as part of #359). Is there some kind of check Core-side forcing us to set this?

Reply:
No, I was confused by something going on in one of the tests and forgot to clean this up